src/test/java/mate/academy/service/impl/UserServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.service.impl;
+
+import java.util.Optional;
+import mate.academy.model.User;
+import mate.academy.service.UserService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class UserServiceImplTest {
+    private UserService userService;
+    private PasswordEncoder passwordEncoder;
+
+    @BeforeEach
+    void SetUp() {
+        userService = Mockito.mock(UserService.class);
+        passwordEncoder = Mockito.mock(PasswordEncoder.class);
+    }
+
+    @Test
+    void save() {
+        User user = getTestUserWithId();
+        Mockito.when(passwordEncoder.encode(Mockito.any())).thenReturn("");
```

Comment:
```suggestion
        Mockito.when(passwordEncoder.encode(Mockito.any())).thenReturn(user.getPassword());
```

I think we should not use empty password

Reply:
added